src/backend/crew_ai/tasks.py

```
@@ -1,7 +1,9 @@
-from crewai import Task
+from crewai import Task, TaskOutput
 import logging
+import json
+import re
 from pydantic import BaseModel, Field
-from typing import List, Optional
+from typing import List, Optional, Tuple, Any
```

Ruff (0.14.14)

`typing.List` is deprecated, use `list` instead

(UP035)